packages/paste-core/components/side-panel/src/SidePanelBody.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { css, styled } from "@twilio-paste/styling-library";
+import type { ThemeShape } from "@twilio-paste/theme";
+import * as React from "react";
+
+import type { SidePanelBodyProps } from "./types";
+
+// This wrapper applies styles that customize the scrollbar and its track
+const SidePanelBodyWrapper = styled.div(({ theme }: { theme: ThemeShape }) => {
+  const { colorBackgroundBody, colorBackground, colorBackgroundStrong } = theme.backgroundColors;
+
+  return css({
+    // Scrollbar
+    "::-webkit-scrollbar": {
+      width: "6px",
+    },
+    // Track
+    "::-webkit-scrollbar-track": {
+      marginTop: "8px",
+      background: colorBackgroundBody,
+      borderRadius: "10px",
+    },
+    // Handle
+    "::-webkit-scrollbar-thumb": {
+      background: colorBackground,
+      borderRadius: "10px",
+    },
+    // Handle on hover
+    "::-webkit-scrollbar-thumb:hover": {
+      background: colorBackgroundStrong,
+    },
+  });
+});
+
+const SidePanelBody = React.forwardRef<HTMLDivElement, SidePanelBodyProps>(
+  ({ element = "SIDE_PANEL_BODY", children, ...props }, ref) => {
+    return (
+      <Box
+        {...safelySpreadBoxProps(props)}
+        as={SidePanelBodyWrapper as any}
+        width="100%"
+        height="100%"
+        paddingLeft="space40"
+        // @ts-expect-error no 6px spacing token but we need this amount of padding to offset the scrollbar
```

Comment:
question: why didn't we need to do this with Side Modal?

Reply:
<img width="987" alt="image" src="https://github.com/twilio-labs/paste/assets/75342690/d4f94d50-93a8-4b9c-b3f6-f2d1f13fad8e">

Side Modal doesn't have a stylized scrollbar and Side Panel (like Sidebar) does. Side Modal right padding starts after the scrollbar, so the right side of the content of SideModalBody jumps to the left when the content starts to scroll.

I think stylized scrollbar without the jumping is a nicer experience but it's a really small difference and doing it the way Side Modal is built would avoid using the hard-coded px value. If you want to change it to be like Side Modal I'll add it to the refinement ticket